ksqldb-serde/src/main/java/io/confluent/ksql/serde/connect/ConnectDataTranslator.java

```diff
@@ -232,6 +233,12 @@ private Object toKsqlValue(
       case STRING:
         // use String.valueOf to convert various int types and Boolean to string
         return String.valueOf(convertedValue);
+      case BYTES:
```

Comment:
Ok, this looks reasonable, but it also looks like it could change things.  

What all does this interact with?

Reply:
I thought the same. I am not sure why this part didn't wrap the bytes to a ByteBuffer before if KSQL uses ByteBuffer for the BYTES column type. So far, only the `AvroDataTranslator` was failing on this test, but I updated it. I also run some QTT tests and run all formats (avro, json, json-sr, proto, delimited) manually with simple, array, map and struct bytes and none of them failed. So I don't know if this code was just missed when supporting the BYTES type.

Reply:
is this related to `JsonSchemaConverter`? If not, I guess we can separate this out to a different PR. The reason is `JsonSchemaConverter` changes are already complex and probably we don't want to mix with other fixes if not necessary.

Reply:
Again, if this is not related to this PR, we can separate it out to another PR